src/wallet/rpcwallet.cpp

```diff
@@ -3447,6 +3447,69 @@ UniValue z_getaddressforaccount(const UniValue& params, bool fHelp)
     return result;
 }
 
+UniValue z_listaccounts(const UniValue& params, bool fHelp)
+{
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (fHelp || params.size() > 1)
+        throw runtime_error(
+            "z_listaccounts\n"
+            "\nReturns the list of accounts created through z_getnewaccount.\n"
+            "\nResult:\n"
+            "[\n"
+            "   {\n"
+            "     \"account\": \"uint\",           (uint) The account id\n"
+            "        {\n"
```

Comment:
This isn't valid JSON syntax, I think it should be something like `"addresses": [{...}, ...]`.